suite-common/message-system/config/config.v1.json

```diff
@@ -331,7 +260,7 @@
             "conditions": [
                 {
                     "environment": {
-                        "desktop": "<23.9.3",
+                        "desktop": "<24.1.2",
```

Comment:
id of this message has to be changed

Reply:
let's change it to 24.2.. so that all people are on new design

Reply:
Fixed